Automattic/wp-calypso
# Social Logos: Update the package version

This updates the social-logos package version to 2.5.3 to add the Nextdoor logo.

## Testing Instructions



Check the package change. It can also be tested in conjunction with https://github.com/Automattic/jetpack/pull/31537 and check that the logo appears on the Nextdoor button when enabled.

<img width="511" alt="image" src="https://github.com/Automattic/wp-calypso/assets/96462/ed5c035a-f5f8-4081-bfbb-2b33872d1c25">


## Pre-merge Checklist
